src/python/WMCore/JobSplitting/EventAwareLumiBased.py

```diff
@@ -161,9 +163,9 @@ def algorithm(self, *args, **kwargs):
                 lumisInJobInFile = 0
                 updateSplitOnJobStop = False
                 failNextJob = False
-                # If the number of events per lumi is higher than the limit
+                # If estimated job time is higher the job time limt (condor limit)
                 # and it's only one lumi then ditch that lumi
-                if f['avgEvtsPerLumi'] > eventLimit and f['lumiCount'] == 1:
+                if(f['avgEvtsPerLumi'] * timePerEvent) > jobTimeLimit and f['lumiCount'] == 1:
```

Comment:
you missed a blank space in the beginning.
BTW, how about we add the job length as well to the error message on this line https://github.com/ticoann/WMCore/blob/ac9f6654cb85a69c37ba48f15302c6fc31792f3e/src/python/WMCore/JobSplitting/EventAwareLumiBased.py#L254

Reply:
Do you want to print out total number of jobs?